lava-slave/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM baylibre/lava-slave-base:2019.03_stretch
+FROM lavasoftware/lava-dispatcher:2019.07
```

Comment:
Now that we switch to the official images, this patch sould also remove the lava-master-base and lava-slave-base directories.

Reply:
Removed!